### ISSUE TITLE
question: is there an example dataset?

### ISSUE THREAD

**Issue by magland** [NeurophysVis/RasterVis#37]
Do you have an example dataset for getting started? Or perhaps instructions on creating an example from a specific nwb file on DANDI?

**Comment by edeno** [comment]
Yes, the example data is [here](https://github.com/NeurophysVis/RasterVis/tree/master/public/DATA). I haven't written the conversion from NWB but that is why I proposed it as one of the projects for the pre-COSYNE hackathon.
